katalog/cert-manager/cainjector/deploy.yml

```diff
@@ -50,11 +58,29 @@ spec:
             allowPrivilegeEscalation: false
             capabilities:
               drop:
-                - ALL
+              - ALL
             readOnlyRootFilesystem: true
-          resources:
-            requests:
-              cpu: 50m
-              memory: 50Mi
```

Comment:
mmm... is this gone from upstream too? I'd prefer to have requests set if possible

Reply:
It has been removed from upstream. I will keep